examples/nasjob-example-envelopenet.yaml

```diff
@@ -0,0 +1,144 @@
+apiVersion: "kubeflow.org/v1alpha1"
+kind: StudyJob
+metadata:
+  namespace: kubeflow
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: nasenvelopenet
+spec:
+  studyName: nasenvelopenet
+  owner: crd
+  optimizationtype: maximize
+  objectivevaluename: MeanSSS
```

Comment:
Can you explain, please, what does `MeanSSS` mean?

Reply:
MeanSSS is a feature map statistic, which we get after truncated training of the network. It is mean of sum of squared l2-norm.